keygen/CMakeLists.txt

```diff
@@ -5,13 +5,21 @@ project(enftun-keygen
   LANGUAGES C
 )
 
+option(KEYGEN_USE_TPM "Build keygen with TPM support" ON)
+
 include(GNUInstallDirs)
 
 find_package(OpenSSL REQUIRED QUIET 1.1)
 find_package(Jansson REQUIRED)
 find_package(CURL REQUIRED)
 find_package(sodium REQUIRED QUIET 1.0.11)
 
+if(KEYGEN_USE_TPM)
+  add_definitions(-DKEYGEN_USE_TPM)
+  find_package(TSS2 REQUIRED QUIET)
+  find_package(xtt COMPONENTS tpm REQUIRED QUIET 0.11.0)
```

Comment:
Shouldn't this be `find_package(xapgtum-tpm ...)`, not `find_package(xtt ...`).

Reply:
Ignore this one. I now see that is was fixed in the last commit to consolidate `KEYGEN_USE_TPM` and `BUILD_TPM`.

Reply:
I think you may be looking at an old version of this. It had been `find_package(xtt` previously, but I changed it to `xaptum-tpm` (yesterday, I think)

Reply:
Haha!